app/Http/Resources/UserResource.php

```
@@ -0,0 +1,31 @@
+     * Transform the resource into an array.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return [
+            'id'      => $this->id,
+            'name'    => $this->name,
+            'email'   => $this->email,
+            'profile' => [
+                'id'   => $this->profile->id,
```

PHPStan (2.0.3)

Access to an undefined property App\Http\Resources\UserResource::$profile.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)